docs/preservation-package-spec.md

```diff
@@ -0,0 +1,34 @@
+
+- _dspace_handle_-thesis.zip The top-level folder must be a zipped bag named with the
+  thesis's DSpace handle followed by "-thesis.zip"
+  - data/ There must be a folder in the bag named "data", containing all files associated with the thesis
+    - metadata/ There must be a folder in the "data" folder named "metadata", containing exactly one file of the metadata for the thesis
+      - metadata.csv The metadata file must be named "metadata.csv". See [Metadata File section](#metadata-file) below for details on the contents of this file
+    - _last-kerberos-DEG-DEP-YYYY_-thesis.pdf The data folder must at minimum contain the PDF representation of the thesis
+    - _last-kerberos-DEG-DEP-YYYY-supplemental1.txt_ The data folder may additionally contain any other files submitted with the thesis including supplementary files, signature pages, etc
+  - bag-info.txt Per the bagit specfication, the bag must contain a bag-info.txt file with the required contents
+  - bagit.txt Per the bagit specfication, the bag must contain a bagit.txt file with the required contents
+  - manifest-_algorithm_.txt Per the bagit specfication, the bag must contain at least one manifest-_algorithm_.txt file containing the checksum and file path of each file in the data folder, where _algorithm_ in the file name should be the cryptographic hash algorithm use to create the checksums, e.g. "sha256"
+  - tagmanifest-_algorithm_.txt Per the bagit specfication, the bag must contain at least one tagmanifest-_algorithm_.txt file containing the checksum and file path of each file in the top-level bag folder, where _algorithm_ in the file name should be the cryptographic hash algorithm use to create the checksums, e.g. "sha256"
+
+## Metadata File
+
+See file [1721.1_123456-thesis.zip/data/metadata/metadata.csv](1721.1_123456-thesis.zip/data/metadata/metadata.csv) in this project's docs folder for an metadata file structured according to these requirements.
+
+All metadata about the thesis must be in a single CSV file with a column for each metadata field and a row for each file associated with the thesis.
+
+Thesis-level metadata  (title, abstract, author, etc.) should be in the row for the thesis PDF file. Repeated fields must have a column for each instance of the field in the metadata. Note: If a rights field includes the copyright character, it must be UTF-8 encoded in the CSV file for Archivematica to parse it correctly.
+
+Other files included in the CSV should have blank values for any fields not directly related to the file -- fields associated with the thesis such as title, abstract, author should be blank, but fields specific to the file such as BitstreamChecksumValue should have the value present in the row for that file.
+
+In addition to the metadata submitted to DSpace, some additional fields are required for presentation and should each have their own column in the CSV file. These are:
```

Comment:
Is `presentation` supposed to be `preservation`?

Reply:
Yes, definitely.